packages/extension-chakra-storefront/src/theme/components/project/footer.js

```diff
@@ -76,9 +84,6 @@ export default {
             fontSize: 'sm',
             marginBottom: 6,
             color: 'gray.50'
-        },
-        socialIcons: {
```

Comment:
This dictates the socialIcon Component container styling, we should move it to correct component